rtp/lib/membrane_demo_rtp/receive_pipeline.ex

```diff
@@ -37,21 +37,26 @@ defmodule Membrane.Demo.RTP.ReceivePipeline do
   end
 
   @impl true
-  def handle_notification({:new_rtp_stream, ssrc, 96}, :rtp, _ctx, state) do
+  def handle_notification({:new_rtp_stream, ssrc, 96, _extensions}, :rtp, _ctx, state) do
     state = Map.put(state, :video, ssrc)
     actions = handle_stream(state)
     {{:ok, actions}, state}
   end
 
   @impl true
-  def handle_notification({:new_rtp_stream, ssrc, 120}, :rtp, _ctx, state) do
+  def handle_notification({:new_rtp_stream, ssrc, 120, _extensions}, :rtp, _ctx, state) do
     state = Map.put(state, :audio, ssrc)
     actions = handle_stream(state)
     {{:ok, actions}, state}
   end
 
   @impl true
-  def handle_notification({:new_rtp_stream, _ssrc, encoding_name}, :rtp, _ctx, _state) do
+  def handle_notification(
+        {:new_rtp_stream, _ssrc, encoding_name, _extensions},
+        :rtp,
+        _ctx,
+        _state
+      ) do
     raise "Unsupported encoding: #{inspect(encoding_name)}"
   end
 
```

Comment:
If I were you I would not ignore the unhandled notifications (the line below) :P

Reply:
There are some notifications from UDP sources, but I agree it's a better idea to match on them and log info that audio and video UDP sources were connected.